src/app/manage-users/manage-users.module.ts

```diff
@@ -0,0 +1,28 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { AllUsersComponent } from './all-users/all-users.component';
+import { ManageUserDetailsComponent } from './manage-user-details/manage-user-details.component';
+
+import { RouterModule, Routes } from '@angular/router';
+import { SharedModule } from 'src/app/shared/shared.module';
+
+const manageRoutes: Routes = [
+  {
+    path: 'Manage', component: AllUsersComponent,
```

Comment:
`Manage` route should be lower-case `manage`

Reply:
changed to lower case as manage-users